app/src/main/cpp/DeviceUtils.h

```diff
@@ -9,12 +9,19 @@
 #include "vrb/MacroUtils.h"
 #include "vrb/Forward.h"
 #include "vrb/Geometry.h"
+#include "vrb/Vector.h"
 #include "Device.h"
 
 #include <unordered_map>
 
 namespace crow {
 
+#if defined(HVR)
+  const vrb::Vector kAverageHeight(0.0f, 1.6f, 0.0f);
+#else
+  const vrb::Vector kAverageHeight(0.0f, 1.7f, 0.0f);
```

Comment:
I know this is from the original code, but I don't think this distinction makes any sense

Reply:
yeah, actually I want to get rid of that at some point, we depend a lot on that and we should not at all